src/components/charts/line.js

```diff
@@ -16,6 +16,7 @@ import {
 
 import {
   modes,
+	modesDescription,
```

Comment:
Could you, please, fix this irregular space and rename the variable to "tooltips"?